### PR TITLE
Add data attributes to cookie banner

### LINKED ISSUE #4148: Event tracking - data attributes for cookie banner
**Is your feature request related to a problem? Please describe.**
Add data attributes for event tracking in the cookie banner component.

**Describe the solution you'd like**
Add data attributes to the cookie banner component which will be used for tracking the component in Simorgh, e.g. `data-cookie-banner=accept`.

Data attributes should be added for the overall component and for the links the users will click on, i.e. accept, reject.

**Testing notes**
[Tester to complete]

Dev insight: Snapshots

- [ ] This feature is expected to need manual testing.

**Additional context**
* https://paper.dropbox.com/doc/Event-tracking--AjZNWSURmJ_~rHV8QUSJkDShAg-3i47TvVb9IJBMJFcL8D6u#:uid=075342290973197664698314&h2=Cookie-banner
* WSFRONTPGS-177
* https://github.com/bbc/simorgh/tree/latest/src/app/containers/ATIAnalytics

### PR DESCRIPTION
**Pull request by ghost**
Resolves #4148

**Overall change:** _Add data attribute to cookie banner to allow for event tracking._

**Code changes:**

- _Add `data-cookie-banner` attribute with `reject and `accept` as values._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/ConsentBanner/Banner/index.amp.jsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { bool, string } from 'prop-types';
 import { ConsentBanner } from '@bbc/psammead-consent-banner';
 import { ServiceContext } from '#contexts/ServiceContext';
 import BannerText from './Text';
+import getDataAttribute from './getDataAttribute';
 
 const Accept = (message, onClick) => (
```
Comment (pharingee) [inline src/app/containers/ConsentBanner/Banner/index.amp.jsx]: It seems you skipped adding `dataAttribute` to the amp Banner.

Comment (pharingee) [inline src/app/containers/ConsentBanner/Banner/index.amp.jsx]: It seems you skipped adding dataAttribute to the amp Banner.

**Review by pharingee — CHANGES_REQUESTED** [review]

File: src/app/containers/ConsentBanner/Banner/index.amp.jsx (hunk 1 above)
Comment (ghost) [reply src/app/containers/ConsentBanner/Banner/index.amp.jsx]: Added data attributes to amp Banner.

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by j-pendlebury — APPROVED** [review]

**Review by pharingee — APPROVED** [review]
LGTM! 👍🏾

**Pull request merged**